CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Flipper Room: The Morph Ball Launcher now connects to the door to Navigation Station and is trivial, instead ot connecting to the door to Elun Transport Access and requiring that the Flipper has been rotated.
 - Changed: Flipper Room: Spinning the Flipper with the ledge warp from below now connects from the door to Elun Transport Access instead of from the Tunnel to Spider Magnet Elevator. It is now also logical to perform the ledge warp from below when Transport Randomizer is disabled.
 
+##### Ferenia
+
+- Fixed: Updated logic from Underwater Bottom in Space Jump Room to include more ways to reach Space Jump Room Access.
```

Comment:
If you just include more ways, thats an "added: "
"Fixed: " is if you change something existing that was broken/faulty